frontend/src/app/components/project-detail/project-detail.component.html

```diff
@@ -0,0 +1,17 @@
+<div class="centered">
+  <div *ngIf="project">
+    <div>
+      <label>GitHub repo: <a href="{{ project.url }}">{{ project.url }}</a></label>
+    </div>
+    <div>
+      <label>Keyboard id: <input [(ngModel)]="project.name" placeholder="name"/></label>
+    </div>
+    <button (click)="createPullRequest()">Create pull request</button>
+  </div>
+  <div *ngIf="!project">
+    <div class="spinner">
+      <fa-icon icon="spinner" spin="true" size="5x"></fa-icon>
```

Comment:
It's been a few years since I've looked at Font Awesome.

I thought the syntax from https://fontawesome.com/icons/spinner?style=solid would be something like
```
<i class="fas fa-spinner"></i>
```

Reply:
`fa-icon` is the special Angular syntax (https://www.npmjs.com/package/@fortawesome/angular-fontawesome)